README.md

```diff
@@ -1,38 +1,57 @@
-# create-svelte
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder)
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte);
+# Good First Issue Finder by EddieHub
 
-## Creating a project
+Good First Issue Finder helps new open source contributors pave their path into the world of open source through good first issues
 
-If you're seeing this, you've probably already done this step. Congrats!
+![eddiehub issue finder](https://user-images.githubusercontent.com/91655303/159216752-c794b00c-226e-40b1-b626-160878e4f73a.png)
 
-```bash
-# create a new project in the current directory
-npm init svelte@next
+<!-- (Can be only done after merged 😅)
+## Example using Gitpod, ephemeral dev environment in the cloud (free)
 
-# create a new project in my-app
-npm init svelte@next my-app
-```
+![Gitpod GIF with progress bar](https://user-images.githubusercontent.com/46727048/146048451-ed4ff31a-c178-4713-a9e0-95118be742dc.gif)
+-->
 
-> Note: the `@next` is temporary
+## 👨‍💻 Demo
```

Comment:
I'm not sure if `Demo` fits here. Its the live website which will also be used to find the issues so for me its not only a demo website rather than the productive environment.